packages/terra-profile-image/src/ProfileImage.jsx

```diff
@@ -42,30 +50,71 @@ const propTypes = {
 
 const defaultProps = {
   fit: 'cover',
+  height: '75',
+  width: '75',
+  variant: 'default',
 };
 
 const isOnlyNumbers = toTest => !(/\D/).test(toTest);
 
 const ProfileImage = (props) => {
+  const {
+    src, alt, fit, height, width, variant, onLoad, onError, ...customProps
+  } = props;
   const theme = React.useContext(ThemeContext);
-  // img tags assume a height attribute of only numbers is in px but CSS does not
-  const fixedHeight = isOnlyNumbers(props.height) ? `${props.height}px` : props.height;
-  const fixedWidth = isOnlyNumbers(props.width) ? `${props.width}px` : props.width;
+
+  // Terra-Image uses a height and width attribute using only numbers, the placeholder span needs css in 'px'
+  const placeholderHeight = isOnlyNumbers(height) ? `${height}px` : height;
+  const placeholderWidth = isOnlyNumbers(width) ? `${width}px` : width;
+  const placeholderSize = { height: placeholderHeight, width: placeholderWidth };
+  const placeholderClassNames = classNames(
+    cx([
+      'profile-image',
+      'placeholder',
+      fit,
+      variant,
+      theme.className,
+    ]),
+    customProps.className,
+  );
 
   /* eslint-disable react/forbid-dom-props */
-  const placeholderImage = (
+  const ProfileImagePlaceholder = (
```

Comment:
I'd keep the leading p lowercase, as this isn't defining a component.

Reply:
Updated: https://github.com/cerner/terra-core/pull/3461/commits/a3817dcd871a7d18ab7ebd20bf5f9141273e5f40